.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": "x86-Debug"
+}
```

Comment:
We don't want to commit per-user settings like this. You should roll back this change, and make sure that the `.gitignore` file mentions the `.vs` directory.